hardhat/test/utils/payload.ts

```diff
@@ -0,0 +1,23 @@
+export const paddedHex = (
```

Comment:
what else should go in this file? seems this should just be in another file

Reply:
@crabsinger  removed the file